contracts/DelayManageable.sol

```diff
@@ -23,10 +22,16 @@ contract DelayManageable is Pausable, GovManager, DelayEvents, DelayModifiers {
         uint256[] memory _amounts,
         uint256[] memory _minDelays,
         uint256[] memory _cliffTimes
-    ) public onlyOwnerOrGov notZeroAddress(_token) {
-        require(_amounts.length == _minDelays.length, "invalid array length");
-        require(Array.isArrayOrdered(_amounts), "amounts should be ordered");
-        require(Array.isArrayOrdered(_minDelays), "delays should be sorted");
+    )
+        public
+        onlyOwnerOrGov
+        notZeroAddress(_token)
+        sameArrayLength(_amounts.length, _minDelays.length)
+        sameArrayLength(_cliffTimes.length, _minDelays.length)
```

Comment:
About the naming:
It checks if the numbers are the same. we pass the length.
the name suggests we pass the array, and the function does the length.